src/shared/ui/Input/ui/InputIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from "react"
+
+import classNames from "classnames"
+
+interface InputIconProps extends PropsWithChildren {
+  onClick: () => void
+}
+
+export const InputIcon = ({ children, onClick }: InputIconProps) => {
```

Comment:
I would suggest using a different name to abstract it from an input component.
ClickableIcon or PressableIcon

Reply:
Done